### ISSUE TITLE
 Multiple extra artifacts are being published to npm

### ISSUE THREAD

**Issue by nventuro** [OpenZeppelin/openzeppelin-test-helpers#7]
These are the contents of the installed package directory, inside `node_modules`:

```
$ ls
contracts  LICENSE     node_modules  README.md  src   test-integration
index.js   migrations  package.json  scripts    test  truffle-config.js
```

Only `index.js` and `src` should be there, along with the readme and license files.

**Comment by frangio** [comment]
We have to specify the `files` field in package.json.

**Comment by nventuro** [comment]
Fixed in 892a4e220ffd1e1102d74cbcf35c186ccb5aa6b7
